src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -172,25 +219,31 @@ private boolean disconnectLineBothSides(Network network, String lineId) {
                 ));
     }
 
-    public Mono<ElementaryModificationInfos> getElementaryModification(UUID groupUuid, UUID modificationUuid) {
-        return Mono.fromCallable(() -> modificationRepository.getElementaryModification(groupUuid, modificationUuid));
-    }
-
     public Mono<Void> deleteModificationGroup(UUID groupUuid) {
         return Mono.fromRunnable(() -> modificationRepository.deleteModificationGroup(groupUuid));
     }
 
-    private List<ElementaryModificationInfos> doAction(Network network, UUID networkUuid, Runnable modification) {
-        return doAction(network, networkUuid, modification, MODIFICATION_ERROR);
+    private List<EquipmenModificationInfos> doAction(Network network, UUID groupUuid, Runnable modification) {
+        return doAction(network, groupUuid, modification, MODIFICATION_ERROR);
     }
 
-    private List<ElementaryModificationInfos> doAction(Network network, UUID networkUuid, Runnable action, NetworkModificationException.Type typeIfError) {
+    private List<EquipmenModificationInfos> doAction(Network network, UUID groupUuid, Runnable modification, NetworkModificationException.Type typeIfError) {
+        NetworkStoreListener listener = NetworkStoreListener.create(network, groupUuid, modificationRepository);
+        ReporterModel reporter = new ReporterModel("NetworkModification", "Network modification");
```

Comment:
The `ReporterModel` `NetworkModification` must be global for a study
It must be created when the study is created ?

Reply:
An another US